etc/examples/AutoRecover/CleanupTask/bin/fam_cleanup.sh

```diff
@@ -10,14 +10,16 @@ set -e
 sleep 10 # (time to observe the failed tasks in the suite monitor).
 
 # Determine which family member(s) failed, if any
-FAILED_TASKS=$(cylc dump $CYLC_SUITE_NAME | grep $CYLC_TASK_CYCLE_TIME | grep failed | sed -e 's/,.*$//')
+FAILED_TASKS="$(cylc dump "$CYLC_SUITE_NAME" \
+    | grep "$CYLC_TASK_CYCLE_TIME" | grep failed | sed -e 's/,.*$//')"
```

Comment:
Yuck `| grep ... | grep ... | sed ...`! (And maybe we can move away from `cylc dump`? But this is just an example.)

Reply:
I think you can do something like `cylc dump "$CYLC_SUITE_NAME" | awk "/$CYLC_TASK_CYCLE_TIME/ && /failed/" | sed 's/,.*$//'` if you want to get rid of the multiple greps. I don't know that that makes it any better though.

Reply:
I don't think these `etc/examples` scripts see the light of day very often!